proxies/live/apiproxy/targets/apim.xml

```diff
@@ -10,7 +10,7 @@
           <Name>RaiseFault.403Forbidden</Name>
         </Step>
       </Request>
-      <Condition>(proxy.pathsuffix MatchesPath "/_status") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
+      <Condition>(proxy.pathsuffix MatchesPath "/_status") and (proxy.pathsuffix MatchesPath "/_validatormetrics") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
```

Comment:
how can `proxy.pathsuffix` match both `/_status` and `/_validatormetrics`

Reply:
Good spot. It cant. That should be an 'or' in between.